rpc/grpc/server/server.go

```diff
@@ -5,21 +5,29 @@ import (
 	"net"
 	"strings"
 
+	sm "github.com/cometbft/cometbft/state"
+	"github.com/cometbft/cometbft/store"
+
+	v1 "github.com/cometbft/cometbft/proto/tendermint/services/block_results/v1"
```

Comment:
Better call this something else instead of 'v1' because there will be other services, maybe 'pbblkrsltsvc' or something like that